projects/packages/account-protection/src/class-config.php

```diff
@@ -11,9 +11,11 @@
  * Class Config
  */
 class Config {
-	public const TRANSIENT_PREFIX      = 'password_detection';
-	public const ERROR_CODE            = 'password_detection_validation_error';
-	public const ERROR_MESSAGE         = 'Password validation failed.';
-	public const EMAIL_SENT_EXPIRATION = 600; // 10 minutes
-	public const MAX_RESEND_ATTEMPTS   = 3;
+	public const PASSWORD_DETECTION_TRANSIENT_PREFIX      = 'password_detection';
+	public const PASSWORD_DETECTION_ERROR_CODE            = 'password_detection_validation_error';
+	public const PASSWORD_DETECTION_ERROR_MESSAGE         = 'Password validation failed.';
+	public const PASSWORD_DETECTION_EMAIL_SENT_EXPIRATION = 600; // 10 minutes
+	public const PASSWORD_DETECTION_MAX_RESEND_ATTEMPTS   = 3;
+
+	public const VALIDATION_SERVICE_USER_META_KEY = 'jetpack_account_protection_recent_password_hashes';
```

Comment:
This might be named more accurately as `VALIDATION_SERVICE_RECENT_PASSWORDS_USER_META_KEY` (or just `RECENT_PASSWORDS_USER_META_KEY`), in case we want to add additional user meta fields for the validation service in the future.

Reply:
Done in [c9f5d32](https://github.com/Automattic/jetpack/pull/41401/commits/c9f5d32ddc5bfef69748aaf0a44c2d3718367e46) - kept the prefix for now as I add a bunch more class specific constants in the following PR.